service/src/main/resources/migrations/v12__backfila.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE run_instances
```

Comment:
looks good

Reply:
Yeah and tests work. Should be safe back to 5.6.

Reply:
mysql 5.6? who still uses that? :)